packages/sdk/src/oracle/client.ts

```diff
@@ -26,10 +31,14 @@ export class Client {
     const identifier = params.identifier.toLowerCase();
     const chainId = Number(params.chainId);
     const timestamp = Number(params.timestamp);
-    return this.sm.types.setActiveRequest.create({ requester, ancillaryData, identifier, chainId, timestamp });
+    const result = this.sm.types.setActiveRequest.create({ requester, ancillaryData, identifier, chainId, timestamp });
+    this.sm.types.updateActiveRequest.create(undefined);
```

Comment:
I don't completely understand what's going on here, but this seems a little odd? Why do we create or update the active request with undefined? That seems like either a no-op or a delete?

Reply:
a couple things here. Having to pass undefined is a sideeffect of typescript and the way I'm exporting these parameter arguments to create a template.  The `updateActiveRequest` command takes no arguments because it pulls the active request from global state.  You can see here is the file that exports the "template" https://github.com/UMAprotocol/protocol/pull/3886/files#diff-20099ec61981af2c2ad5d2f6b9aecc64715656e69ca5f6552e3f4f5f6079ebd0R7 and the "parameters" are undefined.

Typescript will fail to compile if i leave this blank when calling "create", so i have to explicitly pass undefined.